docs/_resources/changelog.md

```diff
@@ -98,6 +98,10 @@ Legend:
 🔍 Optimization engine
```

Comment:
Should we call out that we are deprecating our support of the Log analytics agent in a future release?

Reply:
Agree. See latest commit.